docs/components/file/file-input-demo.js

```diff
@@ -2,11 +2,11 @@ import Util from '../../_util';
 
 (function () {
     const ICONS = [ 'upload', 'paperclip' ];
-    
+
     const VARIANTS = [
-        { label: 'Primary', val: 'hxPrimary' },
-        { label: 'Secondary', val: '', default: true },
-        { label: 'Tertiary', val: 'hxTertiary' },
+        { label: 'Primary', val: 'hxBtn hxPrimary' },
+        { label: 'Secondary', val: 'hxBtn', default: true },
+        { label: 'Tertiary', val: 'hxBtn hxTertiary' },
```

Comment:
Need to look into reducing repetitive `.hxBtn` class from refactored implementation, before the `v0.17.0` release.

Reply:
do we want to do this in this p/r or no?

Reply:
not right now, but we do need a fast follow